index.html

```diff
@@ -0,0 +1,269 @@
+            </div>
+
+            <div class="main-content">
+                <h2>Multi-Post Stories</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul>
+                    <li>CSS</li>
+                    <li>HTML</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <button>See project</button>
+            </div>
+
+        </div>
+
+        <div class="project-container">
+
+            <div class="showcase">
+                <div class="showcase-image">
+                   <img src="images/Img Placeholder (4).png" alt="">
+                </div>
+
+                <div class="showcase-detail">
+                  <h1>Profesional Art Printing Data</h1>
+                  <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                  <ul>
+                      <li>HTML</li>
+                      <li>Bootstrap</li>
+                      <li>Ruby</li>
+                  </ul>
+              </div>
+                <button>See project</button>
+            </div>
+            <div class="showcase">
+                <div class="showcase-image">
+                   <img src="images/Img Placeholder (4).png" alt="">
+                </div>
+
+                <div class="showcase-detail">
+                  <h1>Profesional Art Printing Data</h1>
+                  <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                  <ul>
+                      <li>HTML</li>
+                      <li>Bootstrap</li>
+                      <li>Ruby</li>
+                  </ul>
+              </div>
+                <button>See project</button>
+            </div>
+            <div class="showcase">
+                <div class="showcase-image">
+                   <img src="images/Img Placeholder (4).png" alt="">
+                </div>
+
+                <div class="showcase-detail">
+                  <h1>Profesional Art Printing Data</h1>
+                  <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                  <ul>
+                      <li>HTML</li>
+                      <li>Bootstrap</li>
+                      <li>Ruby</li>
+                  </ul>
+              </div>
+                <button>See project</button>
+            </div>
+            <div class="showcase">
+                <div class="showcase-image">
+                   <img src="images/Img Placeholder (4).png" alt="">
+                </div>
+
+                <div class="showcase-detail">
+                  <h1>Profesional Art Printing Data</h1>
```

Comment:
- Kindly you will have one `h1` element per page, so use other heading elements for this for better semantic arrangement 👍